process_request/tests.py

```diff
@@ -197,6 +197,12 @@ def test_get_size(self):
             instance = json_from_fixture(fixture)
             self.assertEqual(get_size(instance), size)
 
+        instance = json_from_fixture("instances_error.json")
+        error_message = "Error parsing instances"
+        with self.assertRaises(Exception) as e:
+            get_size(instance)
+        self.assertEqual(str(e.exception), error_message)
+
```

Comment:
I'm not sure it's necessary to test the exception message here, since it's hardcoded in the method and we're just testing that our two hardcoded strings match. I'd be inclined to just do away with lines 200 and 204.

Reply:
I assume you mean line 201, but yeah, easy enough to change.